microsoft-edge/devtools-guide-chromium/whats-new/2021/10/devtools.md

```diff
@@ -0,0 +1,98 @@
+
+In Microsoft Edge version 95, this issue has been fixed.  Selecting the **User agent client hints** button by using the keyboard now expands a form, in which you can specify client hints for the custom device.
+
+To learn more about User-Agent Client Hints, see [Detecting Microsoft Edge from your website](../../../../web-platform/user-agent-guidance.md#user-agent-client-hints).
+
+:::image type="content" source="../../media/2021/10/keyboard-define-ua-client-hints.png" alt-text="Description" lightbox="../../media/2021/10/keyboard-define-ua-client-hints.png":::
+
+To view the history of this feature in the Chromium open-source project, see [Issue 1243827: User agent client hints form accessibility](https://bugs.chromium.org/p/chromium/issues/detail?id=1243827).
+
+
+<!-- ====================================================================== -->
+## Console filters now display grouped messages if the filter matches the group title
+
+<!-- Title: Improved filtering for grouped messages in the Console -->
+<!-- Subtitle: Filters in the Console is now more intuitive, displaying grouped messages only when the filter matches the group label. -->
+
+In earlier versions of Microsoft Edge, there were several problems when trying to filter grouped messages in the Console. You can use `console.group()` to label a group of messages and provide some organization for Console messages. However, when filtering to a group label, the Console wouldn't display individual messages inside the group.  Also, filtering wouldn't hide group labels, even if the label didn't match the filter.
+
+In Microsoft Edge version 95, these problems have been fixed.  Filtering to a label now displays the group label and the individual messages inside of the group.  When the filter doesn't match the group label, the entire group is hidden.
+
+To learn more about the `group()` API for the Console in DevTools, navigate to [Console API reference: group](../../../../devtools-guide-chromium/console/api.md#group).
+
+When the filter string matches a group label, the group and its members are displayed:
+
+:::image type="content" source="../../media/2021/10/filter-matches-group-label.png" alt-text="When the filter string matches a group label, the group and its members are displayed." lightbox="../../media/2021/10/filter-matches-group-label.png":::
+
+When the filter string doesn't match a group label, neither the group nor its members are displayed:
+
+:::image type="content" source="../../media/2021/10/filter-matches-group-label-asdf.png" alt-text="When the filter string doesn't match a group label, neither the group nor its members are displayed." lightbox="../../media/2021/10/filter-matches-group-label-asdf.png":::
+
+To view the history of this feature in the Chromium open-source project, see [Issue 363796: ☂ Console filter inappropriately hides grouped content and doesn't hide group titles](https://bugs.chromium.org/p/chromium/issues/detail?id=363796).
+
+
```

Comment:
fyi, a [draft of four selected Chromium items](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/10/devtools?branch=user%2Fv-mhoffman%2Fchromium-whats-new-95#announcements-from-the-chromium-project), presently without Edge-based captures yet.